CI/integration_tests/test_example01.py

```diff
@@ -0,0 +1,187 @@
+"""
```

Comment:
With things like tests I find doc strings to be quite helpful, particularly if we put the code through a difficult test where we really try to get it to fail. It can help to have a small discussion about what you're doing and what the plan is to push to code to failure.